include/dlaf/util_string.h

```diff
@@ -0,0 +1,23 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <algorithm>
+#include <cctype>
+#include <string>
+
+namespace dlaf::util {
+inline std::string copy_to_lower(const std::string& str) {
+  std::string s(str);
```

Comment:
very minor consideration

```suggestion
inline std::string copy_to_lower(std::string s) {
```